qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -183,7 +185,9 @@ def run(self, dag):
             for node in front_layer:
                 if len(node.qargs) == 2:
                     v0, v1 = node.qargs
-                    if self.coupling_map.graph.has_edge(current_layout[v0], current_layout[v1]):
+                    if self.coupling_map.graph.has_edge(
+                        current_layout._v2p[v0], current_layout._v2p[v1]
+                    ):
```

Comment:
This seems like a reasonable place to access `_v2p` directly (because it's deep in a nested loop, and `current_layout` is being updated after each layer so moving to a single access might not be feasible). Can you add a comment explaining?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7036/commits/95caec850a9ee4bf964a8f6fb1be3aafe774f971